nodejs/week3/meal_sharing_app/src/backend/api/meals.js

```diff
@@ -0,0 +1,142 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const knex = require("../database");
+
+app.use(express.json());
+app.use(express.urlencoded({
+  extended: true
+}));
+
+//GET api/meals/ query parameters
+
+router.get("/", async function (req, res) {
+  const {
+    maxPrice,
+    title,
+    availableReservations,
+    createdAfter,
+    limit
+  } = req.query;
+  const mealTitles = await knex.select("*").table("meal");
```

Comment:
The name here is a little misleading. It is not meal titles but just meals. Its an array of objects. Not array of strings. Because of the `.select("*")`